test/jasmine/tests/sankey_test.js

```diff
@@ -31,7 +32,7 @@ describe('sankey tests', function() {
         var traceOut = { visible: true },
             defaultColor = '#444';
 
-        Sankey.supplyDefaults(traceIn, traceOut, defaultColor, layout);
+        Sankey.supplyDefaults(traceIn, traceOut, defaultColor, Lib.extendFlat({colorway: defaultColors}, layout));
```

Comment:
Might be an overly cautious idea, how about using non-default colors for `colorway`here, or just altering a `sankey` mock to lock in a visible difference.

Reply:
Good call. Actually, given that `('color/attributes').defaults` no longer appears explicitly in the sankey code, seems to me that an end-to-end test of the full default-color case may be the most useful of all, so I just stripped explicit colors out of the dark mock -> https://github.com/plotly/plotly.js/pull/2277/commits/f2ebc7908d2abf26abdfd02e1eeb882310dcd707

Reply:
Yes, makes sense, and works great with a custom `colorway` too!